layouts/New/Blog.tsx

```diff
@@ -34,16 +36,28 @@ const BlogLayout: FC = async () => {
     }));
 
   const blogData = await getBlogCategory(pathname);
+  // haky way to determine the rss feed
+  const rssFeed = () => {
+    if (blogData.category === 'vulnerability') return 'vulnerability';
+    if (blogData.category === 'release') return 'releases';
+    return 'blog';
+  };
```

Comment:
ah, I commented before I saw this code. yes, we should improve upon this. not sure why `announcements` doesnt work but I assume it comes from https://github.com/nodejs/nodejs.org/blob/56799569c7950ab0fe8f69c56364719a4aabfc47/next-data/generators/websiteFeeds.mjs

Reply:
I didn't get why do you meant by "doesn't work" because when I do `curl https://nodejs-org-git-featrss-visible-link-openjs.vercel.app/en/feed/blog.xml` I have the feed. and when I click on link I also have correct link.

But I allow this code snipette isn't sexy but idk how to do better

Reply:
i had update code to use site config.

Reply:
> ah, I commented before I saw this code. yes, we should improve upon this. not sure why announcements doesnt work but I assume it comes from [5679956/next-data/generators/websiteFeeds.mjs](https://github.com/nodejs/nodejs.org/blob/56799569c7950ab0fe8f69c56364719a4aabfc47/next-data/generators/websiteFeeds.mjs)

There's no announcements RSS feed on site config